demos/common/cpp/utils/include/utils/images_capture.h

```diff
@@ -14,6 +14,7 @@ class ImagesCapture {
 
     ImagesCapture(bool loop) : loop{loop} {}
     virtual double fps() const = 0;
+    virtual std::string type() const = 0;
```

Comment:
Why was this added? As far as I can tell, it's not used anywhere.

Reply:
This was added to be consistent with `get_type()` in `images_capture.py`. C++ demos do not need info about the input type yet, but it may be useful for the new demos.

Reply:
I don't think we should add functionality before we actually need it (and frankly, I'm not a big fan of this being in Python demos either, so I'll argue for finding a better solution even if we do need it).

Reply:
Okay, I deleted this function.